### PR TITLE
With love ♥️

### PR DESCRIPTION
**Pull request by danielepolencic**

### CODE REVIEW

File: src/inputHandler.ts [hunk 1]
```diff
@@ -31,9 +31,13 @@ interface sqsRepoMessage {
   installation_id: number
 }
 
-export async function httpQueryRepoHandler (event: APIGatewayProxyEventV2): Promise<APIGatewayProxyResultV2> {
+export async function httpQueryRepoHandler(event: APIGatewayProxyEventV2): Promise<string | undefined> {
+  if (!event.pathParameters) {
+    throw 'Not sure what to do here.'
```
Comment (chrisns) [inline src/inputHandler.ts:36]: this shouldn't be possible, lambda+api gateway routing shouldn't get it here without the path parameters, it'll also probably always be an array even if empty

File: src/inputHandler.ts [hunk 2]
```diff
@@ -43,11 +47,12 @@ export async function httpQueryRepoHandler (event: APIGatewayProxyEventV2): Prom
       QueueUrl: process.env.REPO_QUEUE ? process.env.REPO_QUEUE : 'error',
     })
     .promise()
+  return response.MessageId
```
Comment (chrisns) [inline src/inputHandler.ts:50]: is there a harm in returning the full response?

File: src/inputHandler.ts (hunk 1 above)
Comment (danielepolencic) [reply src/inputHandler.ts:36]: The types suggest that you can. If you say it shouldn't be, just use `event.pathParameters!.something`

File: src/inputHandler.ts (hunk 2 above)
Comment (danielepolencic) [reply src/inputHandler.ts:50]: The types did not match when I installed them. I wasn't sure what you wanted so I picked a random property.

**Pull request merged**